### PR TITLE
[wave2-809] explore programs  Global: Text Hero CTA fpp

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php if ($logged_in && $is_admin) { ?><!-- Include file: <?php echo str_replace('-', '=', __FILE__); ?> --><?php } ?>
+
+<?php
+
+$uri = $_SERVER['REQUEST_URI'];
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php]: Should be switched to request_uri() (https://api.drupal.org/api/drupal/includes!bootstrap.inc/function/request_uri/7.x) That way if we ever changed engines (to nginx) the code would still produce a result.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+if ($uri == '/future-grad-students/explore-programs') {
+  $headingtag = 'p';
+}
+
+?>
+
+<div class="intro-brick <?php echo $classes ?>">
+    <div class="ib-first">
+      <<?php echo $headingtag ?>>
+      <?php echo ($content['title']['#value']); ?>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php]: Should indent this line as it is a child of the heading tag.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php (hunk 1 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php]: thanks @dpfitzsi, that has been updated.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php (hunk 2 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_intro_brick/templates/fieldable-panels-pane--ucgy_gbl_intro_brick.tpl.php]: thanks @dpfitzsi, well spotted. This has been indented now.

**Pull request merged**